jsonschema-validation.xml

```diff
@@ -799,8 +801,9 @@
                     <t>
                         When "if" is absent, or the instance successfully
                         validates against its subschema, validation against
-                        this keyword always succeeds.  Implementations
-                        SHOULD avoid attempting to validate against
+                        this keyword always succeeds, regardless of the
```

Comment:
I know this isn't your change, but I find the phrasing...
"When "if" is absent, or the instance successfully validates against its subschema, validation against this keyword always succeeds..."

This reads to me as, when the subschema of `if` successfully validates, `else` always succeeds [validation], regardless of the actual validation result of the instance against it's subschema.

That can't be the intent...

Reply:
Great feedback, I'll play around with this some more.
The main point of bug-fixing the validation spec is to clarify `if`/`then`/`else` so please highlight everything that's awkward.